src/migration/1626738106321-addSubmissions.ts

```diff
@@ -0,0 +1,45 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class addSubmissions1626738106321 implements MigrationInterface {
+  name = 'addSubmissions1626738106321'
+
+  // prettier-ignore
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE TYPE "submissions_type_enum" AS ENUM('filepath', 'json', 'text')`);
+
+    await queryRunner.query(`CREATE TABLE "submissions" (
+        "id" SERIAL NOT NULL,
+        "created_at" TIMESTAMP NOT NULL DEFAULT now(),
+        "updated_at" TIMESTAMP NOT NULL DEFAULT now(),
+        "deleted_at" TIMESTAMP,
+        "course_id" integer NOT NULL,
+        "assignment_id" integer NOT NULL,
+        "user_id" integer NOT NULL,
+        "submitted_by" integer NOT NULL,
+        "original_submission_id" integer,
+        "type" "submissions_type_enum" NOT NULL,
+        "content" character varying NOT NULL,
+        "submitter_ip" character varying(64) NOT NULL,
+        "original_submission" integer,
```

Comment:
Both 'original_submission_id' and 'original_submission' were created from typo in model